receiver/awscontainerinsightreceiver/internal/k8swindows/k8swindows.go

```diff
@@ -37,36 +38,39 @@ func New(logger *zap.Logger, decorator *stores.K8sDecorator, hostInfo host.Info)
 	if nodeName == "" {
 		return nil, errors.New("missing environment variable HOST_NAME. Please check your deployment YAML config")
 	}
-	k8sSummaryProvider, err := new(logger, hostInfo)
-	if err != nil {
-		logger.Error("failed to initialize kubelet summary provider, ", zap.Error(err))
-		return nil, err
-	}
 
 	metricsExtractors = []extractors.MetricExtractor{}
 	metricsExtractors = append(metricsExtractors, extractors.NewCPUMetricExtractor(logger))
 	metricsExtractors = append(metricsExtractors, extractors.NewMemMetricExtractor(logger))
+
+	ksp, err := kubelet.New(logger, &hostInfo, metricsExtractors)
+	if err != nil {
+		logger.Error("failed to initialize metricProvider summary provider, ", zap.Error(err))
```

Comment:
align the name in the comment with the variable name or class name (or at least close enough that it makes sense)

Reply:
Addressed